templates/4a_compensate_image_data.ipynb

```diff
@@ -38,9 +38,8 @@
     "import shutil\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    output_folder = 'rosetta_output'
</pre><p>Does output_folder need to be user-specified? Seems like cohort_name is the important one right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>